uMenu/romfs/fr.json

```diff
@@ -0,0 +1,132 @@
+{
+    "yes": "Oui",
+    "no": "Non",
+    "ok": "Ok",
+    "cancel": "Annuler",
+    "menu_multiselect": "Sélection multiple",
+    "menu_multiselect_cancel": "La sélection multiple a été annulée.",
+    "menu_rename_folder": "Renommer le dossier",
+    "menu_rename_folder_conf": "Souhaitez-vous renommer ce dossier ?",
+    "swkbd_rename_folder_guide": "Entrez le nouveau nom du dossier",
+    "hb_mode_entries_add": "Souhaitez-vous ajouter toutes les entrées sélectionnées au menu principal ?",
+    "hb_mode_entries_added": "Les entrées sélectionnées ont été ajoutées au menu principal.",
+    "hb_mode_entries_some_added": "Certaines entrées sélectionnées ont été ajoutées dans le menu principal, mais d'autres avaient déjà été ajoutées.",
+    "menu_move_to_folder": "Souhaitez-vous déplacer toutes les entrées sélectionnées dans un nouveau dossier ou dans un dossier déjà existant ?",
+    "menu_move_new_folder": "Nouveau dossier",
+    "menu_move_existing_folder": "Dossier existant",
+    "menu_move_select_folder": "Sélectionnez un dossier vers lequel déplacer le contenu sélectionné.",
+    "menu_move_select_folder_cancel": "Sélection de dossier annulée.",
+    "menu_move_existing_folder_conf": "Souhaitez-vous déplacer toutes les entrées sélectionnées dans ce dossier ?",
+    "swkbd_new_folder_guide": "Entrez le nom du dossier",
+    "menu_move_from_folder": "Souhaitez-vous déplacer toutes les entrées sélectionnées vers le menu principal ?",
+    "app_launch_error": "Une erreur s'est produite lors de la tentative de lancement du titre",
+    "entry_options": "Options d'entrée",
+    "entry_action": "Que souhaitez-vous faire avec l'entrée sélectionnée ?",
+    "entry_move": "Déplacer vers/depuis le dossier",
+    "entry_remove": "Supprimer l'entrée",
+    "entry_remove_conf": "Souhaitez-vous vraiment supprimer l'accès à cette entrée du menu principal ?",
+    "entry_remove_ok": "L'entrée a été supprimée avec succès.",
+    "hbmenu_launch": "Lancer le hbmenu",
+    "unknown": "Unconnu",
+    "power_dialog": "Option d'alimentation",
```

Comment:
Options ?

Reply:
Yes, it's for stand by, shut down and more...